File: fc-large1/ecommerce-app/admin/src/main/java/com/fastcampus/ecommerce/admin/repository/CustomerRepository.java
package com.fastcampus.ecommerce.admin.repository;

import com.fastcampus.ecommerce.admin.domain.customer.Customer;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface CustomerRepository extends JpaRepository<Customer, Long> {
    List<Customer> findTop100ByIsDeletedIsFalse();
}

File: graphql/graphqlex/reactive-r2dbc/src/main/kotlin/com/yscorp/reactiver2dbc/domain/CustomerRepository.kt
package com.yscorp.reactiver2dbc.domain

import org.springframework.data.repository.kotlin.CoroutineCrudRepository
import org.springframework.data.repository.reactive.ReactiveCrudRepository
import org.springframework.stereotype.Repository

@Repository
interface CustomerRepository : CoroutineCrudRepository<Customer, Int> {

}

